WordPress/Classes/ViewRelated/Jetpack/Branding/Fullscreen Overlay/JetpackFullscreenOverlayGeneralViewModel.swift

```diff
@@ -287,6 +312,9 @@ private extension JetpackFullscreenOverlayGeneralViewModel {
             static let notificationsContinueButtonTitle = NSLocalizedString("jetpack.fullscreen.overlay.notifications.continue.title",
                                                                             value: "Continue to Notifications",
                                                                             comment: "Title of a button that dismisses an overlay and displays the Notifications screen.")
+            static let continueButtonTitle = NSLocalizedString("jetpack.fullscreen.overlay.phaseThree.general.continue.title",
```

Comment:
[nit] 

```suggestion
            static let continueButtonTitle = NSLocalizedString("jetpack.fullscreen.overlay.general.continue.title",
```

Reply:
I intentionally did not update this to not force a new translation of an already translated string. But now I'm having second thoughts. Not sure what's worse, having an inaccuracy in the key or translating an already translated string. What do you think?
Related thread: p1650995535103199/1650995010.603439-slack-CC7L49W13

Reply:
Thanks for the thread link, that makes a lot of sense.

Hmmm. In this case the renaming is a super minor change (i.e. it doesn't noticeably improve the readability etc), so maybe it's best to leave it as is.